docs/develop/api-reference/index.mdx

```diff
@@ -196,6 +196,93 @@ puts client.caches.to_a.join(", ")
   cli={`momento cache list`}
 />
 
+### Flush cache
+
+Flushes all data from a cache
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
+
+:::
+
+<SdkExamples
+  js={`
+const authToken="eyJhbGc.MyTestToken";
+const defaultTTL = 15;
+const momento = new SimpleCacheClient(authToken, defaultTtl);
```

Comment:
These aren't the arguments for the constructor anymore. 

*Taken from the [npmjs page for the SDK](https://www.npmjs.com/package/@gomomento/sdk)*

```javascript
const credentialsProvider = new EnvMomentoTokenProvider('MOMENTO_AUTH_TOKEN');

const loggerOptions: LoggerOptions = {
  level: LogLevel.INFO,
  format: LogFormat.JSON,
};

const defaultTtl = 60;
const momento = new SimpleCacheClient({
  configuration: Configurations.Laptop.latest(loggerOptions),
  credentialProvider: credentialsProvider,
  defaultTtlSeconds: defaultTtl,
});
```

Reply:
This makes me question how up to date are any of the code examples on this page. @cprice404 any thoughts on how best to have the team audit this?

Reply:
Revised implementation. The example from the SDK docs was not right.

```javascript
const { SimpleCacheClient, EnvMomentoTokenProvider, LogLevel, LogFormat, Configurations } = require('@gomomento/sdk');
const credentials = new StringMomentoTokenProvider ({ authToken: 'MOMENTO_AUTH'});

const momento = new SimpleCacheClient({
  configuration: Configurations.Laptop.latest(),
  credentialProvider: credentials,
  defaultTtlSeconds: 60
});
```

Reply:
OK, added that into the file.